src/client/components/GameCardComponent/FormComponent/FormComponent.style.css

```diff
@@ -0,0 +1,34 @@
+@import url('https://fonts.googleapis.com/css2?family=Rammetto+One&family=Roboto&display=swap');
+
+html {
+  background-color: #fba85c;
+  background-image: url('../../../assets/images/background.png');
+  background-blend-mode: multiply;
+  background-repeat: no-repeat;
+}
+
+.form-input {
+  position: absolute;
+  background: #ffffff;
+  position: relative;
+  margin-top: 37px;
+  width: 335px;
```

Comment:
These two values could / should probably be controlled by the outside wrapping component to make it more responsive and reusable on multiple screen sizes and pages 😃